instrumentation/opentelemetry-instrumentation-elasticsearch/src/opentelemetry/instrumentation/elasticsearch/__init__.py

```diff
@@ -157,9 +157,10 @@ def wrapper(wrapped, _, args, kwargs):
             except Exception as ex:  # pylint: disable=broad-except
                 if span.is_recording():
                     if isinstance(ex, elasticsearch.exceptions.NotFoundError):
```

Comment:
can't we remove this "if" block entirely? the result is always StatusCode.Error.